templates/failed-payment.html

```diff
@@ -1,46 +1,101 @@
 {% extends "base.html" %} {% block main %}
-<h2>Oops! Let's fix this, {{ name }}</h2>
-<div class="error-banner">
-  <svg
-    width="24"
-    height="24"
-    viewBox="0 0 24 24"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
+<h2
+  style="
+    font-size: 50px;
+    font-weight: 300;
+    line-height: 55px;
+    margin: 0;
+    text-align: center;
+    color: black;
+  "
+>
+  Oops! Let's fix this, {{ name }}
```

Comment:
Is name always populated in our system? I could see it being unset sometimes unless we fall back to email or something if they never told us their name

Reply:
this is passed in by the caller. I believe it's `name or username` for the owner atm.